pkg/cloudprovider/providers/gce/cloud/service.go

```diff
@@ -27,6 +27,10 @@ import (
 	ga "google.golang.org/api/compute/v1"
 )
 
+const (
+	maxOperationGetErrorStreak = 50
```

Comment:
I would call this maxConsecutiveOperationGetErrors

Reply:
Sure